addons/netfox.extras/state-machine/network-state.gd

```diff
@@ -0,0 +1,33 @@
+@icon("res://addons/netfox/icons/state-synchronizer.svg")
+extends Node
+class_name NetworkedState
+
+@export var state_machine: NetworkedStateMachine
+
+## A networked state for use with [NetworkedStateMachine].
+
+func _ready():
+	if state_machine == null and get_parent() is NetworkedStateMachine:
+		state_machine = get_parent()
+
+# Custom update callback method
+func update(delta: float, tick: int, is_fresh: bool) -> void:
+	pass
+
+# Custom physics process callback method
+func physics_process(delta: float) -> void:
```

Comment:
I think this is not necessary, the states should be updated as part of the rollback tick loop, which may or may not be running in the physics process, depending on netfox settings. 

But do let me know if I'm missing something!